### PR TITLE
Update defaults for bifrost chart

### PR DESCRIPTION
**Pull request by admiraladmirable**
## Purpose
Make the Bifrost Helm chart easier to run with minimal values. 

Also include a few changes.

## Approach
Summary of default changes:
* Change mount directory to `/mnt/bifrost` to allow `readOnlyRootFilesystem` to be set to true. (default write location is `/tmp/bifrost`
* Pass `--dataDir` and `--stakingDir` by default to point to non root mount directory.
* Unset `storageClass` to allow cluster to auto assign default storage class.

Other changes:
* Disable `livenessProbe` and `readinessProbe`. Need to fix issues with these.
* Incorporate configMap changes from Dion helm chart.
* Mount configMap
* Update range command for `args` and `command` to make yaml look prettier. 

## Testing
Deployed locally. 

Checklist:

* [x] I have bumped the chart version.
* [x] Any new values are backwards compatible and/or have sensible default.

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

File: charts/bifrost/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| args[1] | string | `"/mnt/bifrost/data"` |  |
+| args[2] | string | `"--stakingDir"` |  |
+| args[3] | string | `"/mnt/bifrost/staking"` |  |
+| command | string | `nil` |  |
+| configMap.content | string | `"bifrost {\n  chainReplicator {\n  enableChainReplicator = false\n  checkMissingBlock = true\n  }\n}\n"` |  |
+| configMap.fileName | string | `"application.conf"` |  |
+| configMap.mountPath | string | `"/config/bifrost-dion-config"` |  |
+| image.name | string | `"ghcr.io/topl/bifrost-node"` |  |
+| image.tag | string | `nil` |  |
+| livenessProbe.enabled | bool | `false` |  |
```
Comment (SeanCheatham) [inline charts/bifrost/README.md:21]: We'll probably need to implement actual liveliness/readiness probes in the node at some point

File: charts/bifrost/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+| podSecurityContext.seccompProfile.type | string | `"RuntimeDefault"` |  |
+| ports.p2p | int | `9084` |  |
+| ports.rpc | int | `9085` |  |
+| readinessProbe.enabled | bool | `false` |  |
+| readinessProbe.initialDelaySeconds | int | `30` |  |
+| readinessProbe.periodSeconds | int | `60` |  |
+| readinessProbe.timeoutSeconds | int | `30` |  |
+| replicas | int | `1` |  |
+| resources.limits.cpu | float | `2` |  |
+| resources.limits.memory | string | `"4000Mi"` |  |
```
Comment (SeanCheatham) [inline charts/bifrost/README.md:38]: I think we should aim for request/limit of 2000Mi (or 3000Mi) to start

File: charts/bifrost/values.yaml [hunk 3]
```diff
@@ -50,20 +51,33 @@ ports:
   p2p: 9084
   rpc: 9085
 
+configMap: # Optional
+  # Where the config map should be mounted inside your container's filesystem.
+  mountPath: /config/bifrost-dion-config
```
Comment (SeanCheatham) [inline charts/bifrost/values.yaml:56]: Should this be renamed for tetra?

**Review by SeanCheatham — APPROVED** [review]

File: charts/bifrost/README.md (hunk 2 above)
Comment (admiraladmirable) [reply charts/bifrost/README.md:38]: Updated to 2000Mi.

File: charts/bifrost/values.yaml (hunk 3 above)
Comment (admiraladmirable) [reply charts/bifrost/values.yaml]: updated to be agnostic.

**Pull request merged**